RocketApp/Infrastructure/Sources/NetworkClientExtensions/Loadable.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+struct EquatableVoid: Equatable {}
+
+public enum Loadable<Data: Equatable, Error: Equatable>: Equatable {
```

Comment:
Tohle ma daleko vetsi potencial prepouzitelnosti, takze to sem nepatri. Nevaze se to nutne na `Networking`.

Reply:
Vim, dal bych to do CoreToolkitu ale ve Features na nej nevidim. Uz jsem zkousel vsechno mozny. Adela taky nevi co s tim. Budu ho muset prejmenovat zpatky, uz si nemuzu dovolit na tom palit dalsi cas.

Reply:
Prendat do CoreToolkit.

Reply:
Presunuto.